### DISCUSSION TITLE
apollo2.0.1部署请教

### DISCUSSION THREAD

**Discussion by JobHu** [apolloconfig/apollo#4553] [General]
1、apollo2.0.1部署到阿里云上面，configservice和adminserivce无法注册到eureka上面。
2、apollo2.0.1可以用MAriadb吗？

**Comment by nobodyiam** [comment]
> 2、apollo2.0.1可以用MAriadb吗？

应该是可以的，可以验证下

**Reply by JobHu** [reply]
可以的，谢谢。
